Contentful.NET.Tests/ContentfulClientTests.cs

```diff
@@ -223,6 +224,19 @@ public void GetRequestUrlReturnsLevels()
             Assert.AreEqual(assetId, asset.SystemProperties.Id);
         }
 
+        [Test]
+        public async Task TestCanMFB()
+        {
+            var client = new ContentfulClient(@"1168802daaf1508c9e8e3e41bd114b742a21a0448dbdf1a97251d4b64a8c9685", "xswp1j940j7n", false);
```

Comment:
Would be preferable if we didn't include these keys in source control - in particular *this fork is public*.

Can we:

 - [ ] generate new keys in Contentful and update our usage to use those
 - [ ] figure out a way to run this test without the key being in source control

I'd add that since this project is public it means someone else might want to use it so we should name tests, etc. in a generic way.

Reply:
> - [ ] Generate new keys

https://app.contentful.com/spaces/xswp1j940j7n/api/keys/28JI6UpssYjmbWesJJHlg2

This has been done here.